AppConnectSwift/Babbage.podspec

```diff
@@ -11,13 +10,13 @@ end
 
 Pod::Spec.new do |s|
     s.name               = "Babbage"
-    s.version            = "2016.2.0.84"
+    s.version            = "2016.2.0.100"
```

Comment:
Since this is just a ruby file, could we extract the version number to a constant at the top and then use it here and below in the source?